socorro/processor/processor_app.py

```diff
@@ -11,12 +11,15 @@
 from configman.converters import class_converter
 
 from socorro.app.fetch_transform_save_app import FetchTransformSaveApp, main
-from socorro.external.filesystem.crashstorage import FileSystemRawCrashStorage
 from socorro.external.crashstorage_base import (
   PolyCrashStorage,
   CrashIDNotFound,
 )
 from socorro.lib.util import DotDict
+from socorro.external.fs.fs_new_crash_source import (
+  FSNewCrashSource
+)
+from socorro.external.fs.crashstorage import FSLegacyDatedRadixTreeStorage
```

Comment:
would you consider using some of the more friendly named FS Crash Stores?  Are you storing both raw and processed in the default ```./crashes```.   If so, I suggest using the ```FSDatedPermanentStorage``` class instead.  it's the same class, it just has a friendlier name.